exp/performance.md

```diff
@@ -0,0 +1,52 @@
+The performance of your Anbox Cloud deployment depends on a lot of different factors. To ensure optimal performance, check and monitor all areas and tune your deployment based on your findings.
+
+To measure the performance based on different parameters, you should [run performance benchmarks](https://discourse.ubuntu.com/t/how-to-run-benchmarks/17770). See the provided [Performance benchmarks](https://discourse.ubuntu.com/t/performance-benchmarks/24709) as a reference for what performance you can expect with different hardware configurations.
+
+The main areas for performance tuning are:
+
+- [Container density](#container-density)
+- [Hardware and network setup](#hardware-setup)
+- [Container startup time](#startup-time)
+- [Client devices](#client-devices)
+
+<a name="container-density"></a>
+## Container density
+
+The most apparent performance aspect is how many containers you can run on each of your machines.
+
+Of course, the container density depends a lot on the available hardware. See [About capacity planning](https://discourse.ubuntu.com/t/about-capacity-planning/28717) for detailed information about estimating the necessary capacity and the hardware requirements for your Anbox Cloud deployment.
+
+In addition, check your applications and make sure they use the resources in a fair way. Applications should avoid spikes in GPU utilisation, because such spikes require the application to reserve more resources and therefore reduce the container density.
+
+Generally, applications should use the smallest suitable instance type. However, if you see performance degradation when running the application, using a more powerful instance type usually helps (even though it reduces the container density). As an example, consider an application that runs on an Anbox Cloud deployment that does not have any GPUs installed. In this case, the rendering workload is put on the CPU instead of the GPU, and if the instance type of the application does not have a sufficient number of vCPU cores, the performance of the application is impacted. This can show, for example, in the virtual keyboard being really slow. By switching to a more powerful instance type, the container density is reduced, but the performance of each application container is increased.
```

Comment:
The `performance degradation` may not be the accurate term being used here, because it could be an intermittent issue and the application could perform well after a short-live of time degradation depending on how much the game makes use of the CPU/Memory/GPU. To me,  if an application behaves badly during the entire container runtime with the smallest instance type(meaning the overall poor performance) , considering the bump up instance type then.